### PR TITLE
Add client-go 9.0.0 changelog

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://docs.google.com/spreadsheets/d/1VwVbMA1JBPo30QhX92Z5wuyp5MLUNkW9JGs7s59GZyU/edit#gid=0

/assign @sttts @caesarxuchao

### CODE REVIEW

**Comment by nikhita** [comment]
/hold
until 1.12 is released

**Comment by nikhita** [comment]
oh travis uses an old version of go :/

will send a PR

**Comment by nikhita** [comment]
created https://github.com/kubernetes/client-go/pull/467 to fix travis

**Comment by nikhita** [comment]
Closing and reopening to retrigger travis

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,185 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+
+* Add ability to cancel leader election.
+This also proves useful in integration tests where the whole app is started and
+stopped in each test. ([#57932](https://github.com/kubernetes/kubernetes/pull/57932))
+
+* An example showing how to use fake clients in tests is added.
+([#65291](https://github.com/kubernetes/kubernetes/pull/65291))
+
+* [k8s.io/apimachinery] Create and Update now support `CreateOptions` and `UpdateOptions`.
+([#65256](https://github.com/kubernetes/kubernetes/pull/65256))
```
[the remark this answers is not in the document]
Comment (nikhita) [reply CHANGELOG.md]: > Did you mean #65105?

Yes, got scrambled in the spreadsheet. :)

Fixed. :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,185 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
 because `k8s.io/client-go` depends on them.
 
+# v9.0.0
+
+**Breaking Changes:**
+
+* client-go now supports additional non-alpha-numeric characters in UserInfo
+"extra" data keys. It should be updated in order to properly support extra
+data containing "/" characters or other characters disallowed in HTTP headers.
+([#65799](https://github.com/kubernetes/kubernetes/pull/65799))
```
[the remark this answers is not in the document]
Comment (nikhita) [reply CHANGELOG.md]: > The wording of the release note sounds like it's backward compatible. I'm not sure though.

Yeah, reading it again made me think so too.

Added a new line - `Old clients sending keys which were %-escaped by the user will have their values unescaped by new API servers.`. Ptal if this reads better.

**Comment by nikhita** [comment]
> Though I feel we are abusing your time having you go through all the PRs. 

No problem, it just takes me ~an hour to go through all PRs which touch staging. But yes, automated (https://github.com/kubernetes/test-infra/issues/5843) is nice. :)

**Comment by caesarxuchao** [comment]
Thanks, @nikhita 

/lgtm

**Comment by nikhita** [comment]
/hold

shouldn't 1.12 be released first?

**Comment by nikhita** [comment]
prow is too quick :)

doesn't matter much since 1.12 will be released in a few hours.

**Pull request merged**